samples/Uno.Toolkit.WinUI.Samples/Uno.Toolkit.WinUI.Samples.Mobile/Android/AndroidManifest.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.companyname.Uno.Toolkit.WinUI.Samples" android:versionCode="1" android:versionName="1.0">
```

Comment:
I'm wondering if the package name should be the same as the WinUI Xamarin.Android one (Uno.Toolkit.WinUI.Samples). Similar to how it looks to be done for Gallery

Xamarin:
https://github.com/unoplatform/Uno.Gallery/blob/afb97a5f511ef703d5676382e841f9295a3ba9ba/Uno.Gallery/Uno.Gallery.Droid/Properties/AndroidManifest.xml#L2

net6.0-android:
https://github.com/unoplatform/Uno.Gallery/blob/afb97a5f511ef703d5676382e841f9295a3ba9ba/Uno.Gallery/Uno.Gallery.Mobile/Android/AndroidManifest.xml#L2

Not sure if this really matters though

Reply:
Updated - thanks for noting this